resources/views/staticHeaderPages/aboutUs.blade.php

```diff
@@ -340,15 +340,15 @@ class="orange">{{ trans('aboutus.cycle') }}</span> {{trans('aboutus.seven') }}
     <script src="../../js/aboutUs/script.js"></script>
     <script src="../../js/aboutUs/featherlight.min.js" type="text/javascript"></script>
     <script src="../../js/aboutUs/featherlight.gallery.min.js" type="text/javascript"></script>
+
     <script src="https://cdnjs.cloudflare.com/ajax/libs/vanilla-lazyload/10.4.2/lazyload.min.js"></script>
-    <script src="../dist/lazyload.min.js"></script>
 
     <script type="text/javascript">
-$(document).ready(function () {
-    $(".galleries img").lazyLoad(function () {
-        effect:"fadeIn"
-    });
-});
+        (function() {
+            new LazyLoad({
```

Comment:
Is it still correct? I saw in documentation we should move img src to data attribute. 
Please look at documentation one more time